sympy/core/relational.py

```diff
@@ -1075,9 +1086,96 @@ def _n2(a, b):
             return dif
 
 
+@dispatch(Rational, Pow)
+def _eval_is_ge(rat, rhs): # noqa:F811
+    # i >= b**((+/-1)/j)
+    if rat.is_zero:
+        return  # can resolve based on assumptions of rhs
+    base, exp = rhs.as_base_exp()
+    if base.is_nonnegative:
+        if is_le(exp, S.NegativeOne):
+            return is_ge(base, 1/rat)  # rat >= b^(e<=-1) -> b^(e>=1) >= 1/rat
+        if is_le(base, S.One):
+            return is_ge(rat, S.One)
+
+
+@dispatch(Pow, Rational)
+def _eval_is_ge(lhs, rat): # noqa:F811
+    # b**((+/-1)/j) >= i
+    if rat.is_zero:
+        return  # can resolve based on assumptions of lhs
+    base, exp = lhs.as_base_exp()
+    if base.is_nonnegative:
+        if is_ge(base, S.One) and is_ge(exp, S.One):
+            if is_le(rat, S.One):
+                return True
```

Comment:
So this is `a ** b >= c` and we check `a >= 0` and then `a >= 1` and `b >=1` then if `c <= 1` we return True. I think we only need `b >= 0` here.

Essentially this is:
```
def is_ge(lhs, rat):
    if is_ge(lhs, S.One) and is_le(rat, One):
        return True
```
Maybe it makes sense to write it like that and let a `Pow`, `One` handler take care of further details. Perhaps one of the most important usages of the `is_ge` framework is comparisons against `S.One`...